project-templates/migrations/move-child_name-to-patient_name/ddoc/index.js

```diff
@@ -0,0 +1,28 @@
+
+const path = require('path');
+
+require('dotenv').config({
+  path: path.resolve(__dirname, '..', 'env')
+});
+
+if (!process.env.MIGRATION_NAME) {
+  throw new Error("Missing environment variable: MIGRATION_NAME");
+}
+
+const NAME = process.env.MIGRATION_NAME;
+
+/*
+ * The CouchDB js interpreter by default is pretty old so it does not interpret
+ * ES5/6, so anything inside the ddoc object should be code that works on
+ * netscape navigator.
```

Comment:
Yeah that comment is pretty old.  Here is where I'm putting all CouchDB compatible code for reuse in other projects: https://github.com/medic/medic-api-utils